project.clj

```diff
@@ -16,7 +16,7 @@
   :dependencies [[org.clojure/clojure "1.6.0"]
                  [puppetlabs/trapperkeeper ~tk-version]
                  [puppetlabs/kitchensink ~ks-version]
-                 [puppetlabs/certificate-authority "0.6.0"]
+                 [puppetlabs/ssl-utils "0.8.0"]
                  [puppetlabs/http-client "0.4.0"]
```

Comment:
Note `http-client 0.4.0` is still bringing in `certificate-authority 0.6.0`, but @camlow325 is going to be upgrading this dependency to a newer version that is also on `ssl-utils 0.8.0` soon.